packages/edit-post/src/components/secondary-sidebar/inserter-sidebar.js

```diff
@@ -38,12 +40,25 @@ export default function InserterSidebar() {
 			{ ...inserterDialogProps }
 			className="edit-post-editor__inserter-panel"
 		>
-			<div className="edit-post-editor__inserter-panel-header">
-				<Button
-					icon={ close }
-					onClick={ () => setIsInserterOpened( false ) }
-				/>
-			</div>
+			{ isMobileViewport ? (
+				<VisuallyHidden as="div">
+					<div className="edit-post-editor__inserter-panel-header">
```

Comment:
Can't we just switch between `div` and `VisuallyHidden` here depending on `isMobileViewport`?

Something like `CloseTagName = isMobileViewport ? VisuallyHidden : 'div'`

Reply:
Absolutely, done.